static/src/javascripts.flow.archive/projects/common/modules/identity/sign-in-gate/helper.spec.js

```diff
@@ -38,7 +38,8 @@ jest.mock('@guardian/libs', () => ({
         local: {
             get: jest.fn(() => undefined),
         },
-    }
+    },
+    get: jest.fn(() => undefined),
```

Comment:
Does `get()` exist on `@guardian/libs` ?

I don't see it in the exported functions:

https://github.com/guardian/csnx/blob/f072d9fc97c204f38a9dc5789ca9b7c4c1c1288f/libs/%40guardian/libs/src/index.ts

Reply:
Don’t think it does, why did you add it?

Reply:
@vlkerag I've deleted the flow files here to reduce confusion: https://github.com/guardian/frontend/pull/27027

You will need to rebase these changes, hope its not too much trouble!